src/android/patches/0.18/makePackagePatch.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function makePackagePatch(packageInstance, params) {
+  const processedInstance = packageInstance.replace(
+    /\$\{(\w+)\}/g,
+    (pattern, paramName) => params[paramName]
+      ? `this.getResources().getString(R.strings.${paramName})`
+      : null
+  );
```

Comment:
I don't like the fact this is defined 3x in patches since it does not really change.